src/app/containers/PageContainer/components/DefaultHeaderComponent/DefaultHeaderComponent.tsx

```diff
@@ -506,6 +474,21 @@ export const DefaultHeaderComponent: React.FC = () => {
                             data-action-id="header-lab-perpetuals"
                           />
                         )}
+                        <span className={styles.host}>
+                          {t(translations.mainMenu.origins)}
+                        </span>
+                        <MenuItem
+                          text={t(translations.mainMenu.launchpad)}
+                          className="bp3-popover-dismiss"
+                          onClick={() => history.push('/origins')}
+                          data-action-id="header-origins-link-launchpad"
+                        />
+                        <MenuItem
+                          text={t(translations.mainMenu.claim)}
+                          className="bp3-popover-dismiss"
+                          onClick={() => history.push('/origins/claim')}
+                          data-action-id="header-origins-link-claim"
+                        />
```

Comment:
It looks like Launchpad and Claim of origins was put to be visible only on testnet? I think it should be visible on mainnet